### PR TITLE
Created enemy spawner class

### PR DESCRIPTION
**Pull request by vmontielmg**

### CODE REVIEW

File: Assets/Scripts/ObjectSpawning/EnemySpawner.cs [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    /// <param name="spawner">spawner</param>
+    /// <param name="rooms">list of RoomNode objects</param>
+    /// <param name="difficulty">difficulty of level</param>
+    public EnemySpawner(Spawner spawner, List<RoomNode> rooms, int difficulty)
+    {
+        this.spawner = spawner;
+        this.rooms = rooms;
+        this.difficulty = difficulty;
+        this.indexes = GetRoomIndexes();
+        enemyCount = 0;
```
Comment (Rv-ben) [inline Assets/Scripts/ObjectSpawning/EnemySpawner.cs]: Remember to use `this` keyword

File: Assets/Scripts/ObjectSpawning/EnemySpawner.cs [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemySpawner : MonoBehaviour
+{
+    Spawner spawner;
+    public List<RoomNode> rooms;
+    public int difficulty;
+    public List<int> indexes;
+    static int enemyCount;
```
Comment (Rv-ben) [inline Assets/Scripts/ObjectSpawning/EnemySpawner.cs]: Enemy count should not be static since it is incremented later in the code.

**Review by Rv-ben — CHANGES_REQUESTED** [review]
Looks a lot better, small changes is all that is needed.

File: Assets/Scripts/ObjectSpawning/EnemySpawner.cs (hunk 1 above)
Comment (vmontielmg) [reply Assets/Scripts/ObjectSpawning/EnemySpawner.cs]: done! pushed just now

File: Assets/Scripts/ObjectSpawning/EnemySpawner.cs (hunk 2 above)
Comment (vmontielmg) [reply Assets/Scripts/ObjectSpawning/EnemySpawner.cs]: done x2! thank you

**Pull request merged**